test/OpenSpaceToolkit/Physics/Environment/Gravitational/Earth.test.cpp

```diff
@@ -51,8 +51,7 @@ TEST(OpenSpaceToolkit_Physics_Environment_Gravitational_Earth, Constructor)
         EXPECT_NO_THROW(EarthGravitationalModel earthGravitationalModel(EarthGravitationalModel::Type::EGM96));
         EXPECT_NO_THROW(EarthGravitationalModel earthGravitationalModel(EarthGravitationalModel::Type::EGM2008));
 
-        EarthGravitationalModelManager::Get().setLocalRepository(EarthGravitationalModelManager::DefaultLocalRepository(
-        ));
+        EarthGravitationalModelManager::Get().reset();
```

Comment:
Re: above

Perhaps it's more maintainable to add a `reset()` to the test setup instead of relying on individual tests to clean up after themselves.

Not relevant for this MR though so feel free to close.

Reply:
Agreed. Since this is the only test case that is using the manager I thought it was fine to just reset within the function. But great point!